src/runtime_src/core/edge/drm/zocl/zocl_ctrl_ert.c

```diff
@@ -210,6 +210,9 @@ static int zert_create_scu(struct zocl_ctrl_ert *zert, struct xgq_cmd_config_cu
 {
 	int ret = 0;
 	struct xrt_cu_info info;
```

Comment:
Uninitialized structure.
```suggestion
	struct xrt_cu_info info = {0};
```